src/styles/main.scss

```
@@ -0,0 +1,51 @@
+.watch {
+  position: relative;
+  height: 1080px;
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    z-index: 1;
+    transform: translate(-34%, -50%);
+    width: 30px;
+    height: 30px;
+    background-color: #5f98a8;
+    border-radius: 50%;
+  }
+
+  &__second-hand {
+    position: absolute;
+    bottom: 50%;
+    left: 50%;
+    height: 540px;
```

let's not hardcode height in px, use percents
```suggestion
    height: 50%;
```